mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -1301,10 +1312,9 @@ static LogicalResult populateValidationLogic(
   };
 
   mlir::AllocOp gpuResults;
-  mlir::AllocOp gpuResultsBf16;
   if (operation.getValue() == "conv2d") {
     if (builder.getIntegerType(16) == dataType) {
-      gpuResultsBf16 = getFloatDataFromBF16(outputMemRefCastOp);
+      gpuResults = getFloatDataFromBF16(outputMemRefCastOp);
```

Comment:
logic for backward data and backward filter are missing.

Reply:
fixed